numpy/core/_add_newdocs.py

```diff
@@ -7969,62 +7969,199 @@ def luf(lamdaexpr, *args, **kwargs):
 #
 ##############################################################################
 
-add_newdoc('numpy.core.numerictypes', 'bool_',
-    """NumPy's Boolean type.  Character code: ``?``.  Alias: bool8""")
+def get_type(place, obj):
+    try:
+        return getattr(__import__(place, globals(), {}, [obj]), obj)
+    except Exception:
+        return None
+
+def type_aliases(place, aliases):
+    return [(alias_type, doc) for (alias, doc) in aliases
+                for alias_type in (get_type(place, alias),)
+                if alias_type is not None]
```

Comment:
Rather than using a hack to make a local variable in a comprehension, I think this would be clearer as a generator using `yield`